### ISSUE TITLE
[Bug]: ICE account setting gets reset on every restart

### ISSUE THREAD

**Issue by binarious** [BelledonneCommunications/linphone-desktop#741]
### Context

* Linphone AppImage on Arch Linux intalled from https://aur.archlinux.org/packages/linphone-desktop-appimage.
* Wayland via Sway

### General information

- Device: TUXEDO Pulse
- OS: Arch Linux
- Version of the App: 5.0.10
- Version of the SDK: (Core?) 5.2.19

### Expected behaviour

I expect the application to save the per-account setting of ICE. However the setting gets activated with `stun.linphone.org` on every restart of the application.

### To Reproduce

1. Go to account settings
2. Disable ICE and click ok
3. Close linphone
4. Reopen linphone

### Additional context

I've also disabled ICE in the global network settings.

### SDK logs URL

<details>
  <summary>Logs</summary>

```
[09:29:54:093][0x1d35ee0][Info]"Starting Linphone (bin: linphone)"
[09:29:54:093][0x1d35ee0][Info]"Use locale: de_DE with language: German"
[09:29:54:130][0x1d35ee0][Info]Available fonts :  ("C059", "Cantarell", "D050000L", "DejaVu Math TeX Gyre", "DejaVu Sans", "DejaVu Sans Condensed", "DejaVu Sans Mono", "DejaVu Serif", "DejaVu Serif Condensed", "DejaVuSansMono Nerd Font", "DejaVuSansMono Nerd Font Mono", "Font Awesome 6 Brands", "Font Awesome 6 Free", "FontAwesome", "FreeMono", "FreeSans", "FreeSerif", "icomoon", "Liberation Mono", "Liberation Sans", "Liberation Serif", "MesloLGL Nerd Font", "MesloLGL Nerd Font Mono", "MesloLGLDZ Nerd Font", "MesloLGLDZ Nerd Font Mono", "MesloLGM Nerd Font", "MesloLGM Nerd Font Mono", "MesloLGMDZ Nerd Font", "MesloLGMDZ Nerd Font Mono", "MesloLGS Nerd Font", "MesloLGS Nerd Font Mono", "MesloLGS NF", "MesloLGSDZ Nerd Font", "MesloLGSDZ Nerd Font Mono", "Monospace", "Nimbus Mono PS", "Nimbus Roman", "Nimbus Sans [URW ]", "Nimbus Sans [UKWN]", "Nimbus Sans Narrow", "Noto Color Emoji", "Noto Sans", "Noto Sans UI", "octicons", "P052", "Pomodoro", "PowerlineSymbols", "Sans Serif", "Serif", "Source Code Pro", "Source Code Pro Black", "Source Code Pro ExtraLight", "Source Code Pro Light", "Source Code Pro Medium", "Source Code Pro Semibold", "Source Code Variable", "Standard Symbols PS", "URW Bookman", "URW Gothic", "Z003")
[09:29:54:130][0x1d35ee0][Info]"Running app..."
[09:29:54:191][0x1d35ee0][Info]"Activated selectors:" ("custom", "de_DE", "unix", "linux", "arch")
[09:29:54:192][0x1d35ee0][Info]"Registering types..."
[09:29:54:193][0x1d35ee0][Info]"Registering shared types..."
[09:29:54:194][0x1d35ee0][Info]"Registering tool types..."
[09:29:54:194][0x1d35ee0][Info]"Registering shared tool types..."
[09:29:54:270][0x1d35ee0][Info]"Loading main view..."
[09:29:54:360][0x1d35ee0][Info]"Launch async core creation."
[09:29:54:758][0x1d35ee0][Info]"Set `Friends` path: `/home/redacted/.local/share/linphone/friends.db`"
[09:29:54:766][0x1d35ee0][Info]Core is starting  "Starting up"
[09:29:54:789][0x1d35ee0][Info]Core is running  "On"
[09:29:54:789][0x1d35ee0][Info]Using ZrtpSecrets path :  "/home/redacted/.local/share/linphone/zrtp-secrets.db"
[09:29:54:789][0x1d35ee0][Info]Using UserCertificate path :  "/home/redacted/.local/share/linphone/"
[09:29:54:789][0x1d35ee0][Info]Using RootCa path :  "/tmp/.mount_linphom3yiOm/usr/share/linphone/rootca.pem"
[09:29:54:789][0x1d35ee0][Info]"Start iterate"
[09:29:54:789][0x1d35ee0][Info]Init CoreManager
[09:29:54:797][0x1d35ee0][Info]Sip addresses model from Chats : 3 ms.
[09:29:54:800][0x1d35ee0][Info]Sip addresses model from Calls : 2 ms.
[09:29:54:800][0x1d35ee0][Info]Sip addresses model from Refs : 0 ms.
[09:29:54:800][0x1d35ee0][Info]Sip addresses model from Contacts : 0 ms.
[09:29:54:800][0x1d35ee0][Info]Sip addresses model initialized in: 6 ms.
[09:29:54:809][0x1d35ee0][Info]"Notify event count: 0."
[09:29:54:809][0x1d35ee0][Info]"Notify event count: 0."
[09:29:54:809][0x1d35ee0][Info]"CoreManager initialized"
[09:29:54:844][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterConferences, disambiguation=)"
[09:29:54:849][0x1d35ee0][Debug]"Unable to find a translation. (context=MainWindow, label=mainWindowConferencesTitle, disambiguation=)"
[09:29:54:861][0x1d35ee0][Debug]"Unable to find a translation. (context=MainWindowMenuBar, label=recordings, disambiguation=)"
[09:29:54:864][0x1d35ee0][Debug]"Unable to find a translation. (context=MainWindow, label=newConferenceUriMissing, disambiguation=)"
[09:29:54:864][0x1d35ee0][Debug]"Unable to find a translation. (context=MainWindow, label=newChatRoomUriMissing, disambiguation=)"
[09:29:54:932][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:942][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:946][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:950][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:955][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:961][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:965][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:969][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:972][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:976][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:980][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:984][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:988][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:992][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:54:999][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:003][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:007][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:010][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:014][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:018][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:55:023][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:023][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:023][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyEphemerals, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:024][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAnyChatRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:025][0x1d35ee0][Debug]"Unable to find a translation. (context=Timeline, label=timelineFilterAllSecureLevelRooms, disambiguation=)"
[09:29:55:042][0x1d35ee0][Info]"Open Linphone app."
[09:29:55:042][0x1d35ee0][Info]"Creating subwindow: `qrc:/ui/views/App/Calls/CallsWindow.qml`."
[09:29:55:120][0x1d35ee0][Info]"Subwindow status: `1`."
[09:29:55:149][0x1d35ee0][Info]"Creating subwindow: `qrc:/ui/views/App/Settings/SettingsWindow.qml`."
[09:29:55:183][0x1d35ee0][Info]"Subwindow status: `1`."
[09:29:55:292][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsAdvanced, label=viewlogs, disambiguation=)"
[09:29:55:303][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsUi, label=versionCheckTypeRelease, disambiguation=)"
[09:29:55:303][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsUi, label=versionCheckTypeNightly, disambiguation=)"
[09:29:55:303][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsUi, label=versionCheckTypeCustom, disambiguation=)"
[09:29:55:303][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsUi, label=minimalTimelineFilterTooltip, disambiguation=)"
[09:29:55:303][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsUi, label=minimalTimelineFilterLabel, disambiguation=)"
[09:29:55:331][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsCallsChat, label=AutoDownload, disambiguation=)"
[09:29:55:332][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsCallsChat, label=chatNotificationsEnabledLabel, disambiguation=)"
[09:29:55:333][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsCallsChat, label=callScreenshotEnabledLabel, disambiguation=)"
[09:29:55:334][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoDisplayTitle, disambiguation=)"
[09:29:55:334][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoMosaicLayout, disambiguation=)"
[09:29:55:334][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoActiveSpeakerLayout, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoLayout, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoHybrid, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoOccupyAllSpace, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoBlackBars, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoCallsModeLabel, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoActiveSpeakerModeLabel, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoGridModeLabel, disambiguation=)"
[09:29:55:335][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsVideo, label=videoModeLabel, disambiguation=)"
[09:29:55:357][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsSipAccounts, label=webviewLoginUrlLabel, disambiguation=)"
[09:29:55:357][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsSipAccounts, label=webviewRegistrationUrlLabel, disambiguation=)"
[09:29:55:359][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsSipAccounts, label=defaultDeviceNameLabel, disambiguation=)"
[09:29:55:511][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsCallsChat, label=autoDownloadNever, disambiguation=)"
[09:29:55:511][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsCallsChat, label=autoDownloadAlways, disambiguation=)"
[09:29:55:532][0x1d35ee0][Info]"Notify event count: 0."
[09:29:55:595][0x1d35ee0][Info]"Notify event count: 0."
[09:29:55:602][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:454][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:486][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:506][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:522][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:524][0x1d35ee0][Info]"Notify event count: 0."
[09:29:56:572][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:56:687][0x1d35ee0][Debug]"Unable to find a translation. (context=TimelineItem, label=deleteTimelineTooltip, disambiguation=)"
[09:29:59:292][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsSipAccountsEdit, label=limeServerUrl, disambiguation=)"
[09:29:59:293][0x1d35ee0][Debug]"Unable to find a translation. (context=SettingsSipAccountsEdit, label=videoConferenceURI, disambiguation=)"
[09:30:01:637][0x1d35ee0][Info]"Notify event count: 0."
[09:30:02:850][0x1d35ee0][Info]"Update nat policy."
[09:30:05:929][0x1d35ee0][Warning]Exiting app with the code :  2
[09:30:05:929][0x1d35ee0][Info]"Stopping app..."
[09:30:06:059][0x1d35ee0][Info]"Stop iterate"
[09:30:06:059][0x1d35ee0][Info]"Destroy attached vcard:" VcardModel(0x2458fe0)
[09:30:06:320][0x1d35ee0][Info]"Destroying app..."
```
</details>

**Comment by DirkRaeder-KDO** [comment]
Same issue here in same environment.

It doesn't make any difference whether you empty the STUN server value beforehand or not.

**Comment by julonexus** [comment]
Hi this should be fixed for 5.0.11
Regards
